.github/template/test-setup/action.yaml

```diff
@@ -12,6 +12,10 @@ inputs:
     description: Whether to install the Istio module
     required: true
     default: 'false'
+  operate-in-fips-mode:
+    description: Whether to operate the manager in FIPS mode
```

Comment:
The description here might give the wrong impression that this input determines whether the manager itself runs in FIPS mode or not. Let's change it to something similar to the comment here https://github.com/kyma-project/telemetry-manager/pull/2691/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R114

Reply:
also valid